Palo Alto Networks/paloalto-ngfw/tests/file_cef.json

```diff
@@ -23,6 +23,9 @@
     },
     "destination": {
       "address": "1.1.1.1",
+      "geo": {
+        "name": "1.1.1.1-1.1.1.1"
+      },
```

Comment:
I'm not fond of putting IPs range in `*.geo.name`.
Can you try to put IPs range in a custom field? I think you can check if the field contains `-` and that the two parts are IP addresses.